Documentation/ApiOverview/Events/Events/Core/Imaging/ModifyRecordOverlayIconIdentifierEvent.rst

```diff
@@ -0,0 +1,36 @@
+..  include:: /Includes.rst.txt
+..  index:: Events; ModifyRecordOverlayIconIdentifierEvent
+..  _ModifyRecordOverlayIconIdentifierEvent:
+
+======================================
+ModifyRecordOverlayIconIdentifierEvent
+======================================
+..  versionadded:: 13.0
+
+    The PSR-14 event :php:`ModifyRecordOverlayIconIdentifierEvent` has
+    been introduced which serves as a more flexible replacement for the
+    removed hook:
+
+    *   :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['TYPO3\CMS\Core\Imaging\IconFactory']['overrideIconOverlay']`
+
+The PSR-14 event
+:php:`\TYPO3\CMS\Core\Imaging\Event\ModifyRecordOverlayIconIdentifierEvent` allows
+extension authors to modify the overlay icon identifier of any record icon.
+
+Extensions can listen to the :php:`ModifyRecordOverlayIconIdentifierEvent`
+and perform necessary modifications to the overlay icon identifier
+based on their requirements.
+
+Example
+=======
+
+The corresponding event listener class:
```

Comment:
The event listener class is corresponding to what?

Reply:
This sentence should be omitted like for other events in v13, for example:
https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Events/Events/Backend/AfterBackendPageRenderEvent.html